docs/blog/posts/pydantic.md

```diff
@@ -0,0 +1,235 @@
+---
+draft: False
+date: 2023-12-11
+slug: pydantic
+tags:
+  - pydantic
+authors:
+  - jxnl
+---
+
+# Steering Large Language Models with Pydantic
+
+In the past year, significant progress has been made in utilizing large language models. Prompt engineering, in particular, has gained attention, and new prompting techniques are being developed to guide language models toward specific tasks. While many are building chat bots, an even more exciting application is the generation of structured outputs, whether its extracting structured data, augmenting your RAG application, or even generating
+
+While some have resorted to [threatening human life](https://twitter.com/goodside/status/1657396491676164096?s=20) to generate structured data, we have found that Pydantic even more effective.
+
+In this post, we will explore how we can easily validate structured outputs from language models using Pydantic and OpenAI, to write code that we can trust, then we will introduce a new library called "instructor" that simplifies this process even further while adding additional features as well.
+
+## Pydantic
+
+Unlike libraries like `dataclasses` or `attrs`, `Pydantic` goes a step further and allows you to define a schema for your dataclass. This schema can be used to validate data, but also to generate documentation and even to generate a JSON schema, which is perfect for our use case of generating structured data with language models!
```

Comment:
It could make sense to explain the capability of validation first, then touch on the concept of a schema, just in terms of the significance of the validation capabilities for the below example.

Reply:
do you have an example? do you mean say, "13" -> 13? could you give a small example?

Reply:
<img width="389" alt="Screenshot 2023-12-12 at 10 27 50 PM" src="https://github.com/jxnl/instructor/assets/4852235/29268ecb-42ed-4b3c-91a1-8ea1c69ab17d">

Something like this?